src/java.base/share/classes/java/lang/reflect/code/type/TypeVarRef.java

```diff
@@ -95,12 +96,12 @@ public int hashCode() {
 
     @Override
     public JavaType toBasicType() {
-        throw new UnsupportedOperationException("Type var");
+        return erasure().toBasicType();
```

Comment:
Small changes here, to generalize the behavior of these methods to type-variables, using erasure